vtex/faststore
# Feat: Add `ImageGallery` components

## What's the purpose of this pull request?
`fs/components`
- [x] Moves `ImageGallery` component
- [x] Moves `ImageGallerySelector` component
- [x] Moves `ImageZoom` component

`fs/ui`
- [x] Creates `ImageGallery` styles
- [x] Updates `ImageGallery` styles to follow Figma proposal (e.g. smaller buttons)

`fs/core`
- [x] Review `ImageGallery` structure
- [x] Delete `ImageGallery`'s Storybook and styles files

## How to test it?

run `yarn dev` and check the `ImageGallery` feature on the store's PDP (try searching for the product "Awesome Wooden Bike Sleek") or test it using the doc preview link:
- [ImageGallery](https://faststore-site-git-feat-image-gallery-fs-770-faststore.vercel.app/components/organisms/image-gallery)

### Starters Deploy Preview

Deployments on [starter.store#37](https://github.com/vtex-sites/starter.store/pull/37)
